css/lookbook.scss

```diff
@@ -53,6 +53,39 @@ h1 {
 	object-fit: cover;
 }
 
+
+.lightbox-hidden {
+	display: flex;
+	width: 100%;
+	height: 100%;
+	position: fixed;
+	background-color: rgba(0, 0, 0, 0.8);
+	/*background-color: $offblack;*/
+	top: -100vh;
+	transition: .5s ease;
+	/*opacity: 0.8;*/
+    z-index: 1;
+}
+
+.lightbox {
+	transform: translate(0, 100vh);
```

Comment:
is it necessary that the lightbox slide down? i think having it fade in with a transition on the opacity style would be nice / less jarring maybe?